docs/index.md

```diff
@@ -1,477 +1 @@
-<img src="docs/images/mava.png" width="70%">
-
```

Comment:
Do we not want to landing on the docs to be more detailed than the README anymore?

Reply:
We can def do this going forward. Currently, most of the stuff in the old `index.md` wasn't relevant since we have a new readme and detailed instructions. Unless there is something here we are missing now?